src/main/kotlin/no/nav/plugins/ExceptionHandling.kt

```diff
@@ -11,17 +12,33 @@ class WebStatusException(message: String, val status: HttpStatusCode) : Exceptio
 fun Application.configureExceptionHandling() {
     install(StatusPages) {
         exception<WebStatusException> { cause ->
+            log.error(cause.message)
```

Comment:
Tror jeg ville gjort noe alà;
```suggestion
            log.warn("WebStatusException:${cause.status.value}", cause)
```

Siden vi vet at det bare er vi som kan kaste `WebStatusException` så tenker jeg at det er greit å holde disse på warning-nivå. Også er det ikke nødvendig å kalle `message` når man sender ett exception til loggeren. Istedet lar vi logback håndtere hvordan det skal skrives til loggene.